delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/state/GlobalStateGet.scala

```diff
@@ -0,0 +1,13 @@
+package ch.epfl.bluebrain.nexus.delta.sourcing.state
+
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.EntityType
+import doobie.implicits._
+import doobie.{ConnectionIO, Get, Put}
+
+object GlobalStateGet {
```

Comment:
Wonder if this could be a single method trait and have `GlobalStateStore` implement it? That way the transactor is encapsulated and dependents like `FetchContext` and `FetchActiveOrganisation` can rely on the abstraction (but will be passed the regular `GlobalStateStore` in the production code)

Reply:
The thing is that the store is also relying on other dependencies to be able to stream data (`QueryConfig`) and we don't need it to get a single one.